assets/js/modules/adsense/dashboard/adsense-module-status.js

```diff
@@ -319,10 +319,11 @@ class AdSenseModuleStatus extends Component {
 						/>
 					) }
 
-					{ googlesitekit.canAdsRun && googlesitekit.modules.adsense.setupComplete && accountStatus && (
+					{ googlesitekit.canAdsRun && accountStatus && ( googlesitekit.modules.adsense.setupComplete || 'account-connected' === accountStatus ) && (
```

Comment:
@adamsilverstein Same here, what's the reasoning behind adding the ` || 'account-connected' === accountStatus`?

Reply:
same reasoning as above - we want to show this `AdSenseSetupInstructions` component.  alternately we could adjust `googlesitekit.modules.adsense.setupComplete` on the fly